### DISCUSSION TITLE
download error

### DISCUSSION THREAD

**Discussion by SnufkinHuang** [aclap-dev/video-downloadhelper#644] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://missav.com/dm57/oyc-183",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000001518f84b5c0] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/ca0322b3-83eb-49f4-87ed-7f7626ed7cfd/1280x720/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_de883e53-d25f-4154-8f64-da4faac77244",
    "downloadable_id": "downloadable_2608488763824738"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://missav.com/dm57/oyc-183",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000001518f84b5c0] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/ca0322b3-83eb-49f4-87ed-7f7626ed7cfd/1280x720/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_de883e53-d25f-4154-8f64-da4faac77244",
    "downloadable_id": "downloadable_2608488763824738"
  }
}

**Comment by GrampaWildWilly** [comment]
This has been reported before: #519.  Make sure you click any links that say, "Show x previous replies."
